login_page.py

```diff
@@ -0,0 +1,32 @@
+"""Login page"""
+"""Importing modules"""
+from tkinter import *
+import tkinter.font as font
+from PIL import ImageTk, Image
+
+root=Tk()
+root.geometry("200x200+600+200")
+root.title("Walk2Zero")
+"""Bckground image"""
+img = ImageTk.PhotoImage(Image.open("walk2zeo2.jpg"))
+label_image=Label(root, image=img)
+label_image.place(x=0,y=0,relwidth=1,relheight=1)
+
+"""Top Heading"""
+
+label1= Label(root, text="Hi! I'm E@rth, Happy Walking ", fg='black', font="Head_font",bg="#fcf5cc").place(x=600, y=20)

```

Comment:
Some of the lines (like this one) are a bit too long. PEP 8 recommends a line length of 79 characters. It will be better for readability if we move some of the arguments onto new lines and do the same for other long lines.